src/reducers/systemIntakeReducer.ts

```diff
@@ -149,6 +152,17 @@ function systemIntakeReducer(
           ...prepareSystemIntakeForApp(action.payload)
         }
       };
+    case postIntakeNote.SUCCESS:
+      return {
+        ...state,
+        notes: [
+          ...state.notes,
```

Comment:
If you wanted to make it easier to sort the notes, you could add the new one ot the front of the array instead of the end. We could add a default sort to the GET API so the front end didn't have to do that work. But this works, too, and we can make those refinements later.